.github/workflows/validation.yaml

```diff
@@ -22,6 +22,11 @@ on:
         required: false
         type: string
         default: '--all-files'
+      npmTestScripts:
+        description: A space-delimited list of NPM scripts to run as tests. E.g. 'test e2e'
+        required: false
```

Comment:
`required: false` makes this a non breaking change, right? Should make it so that any repository currently referencing `@main` in their github workflow won't break once this PR is merged to main.

Reply:
Yes correct.